x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -375,57 +375,49 @@ export const EndpointList = () => {
   ]);
 
   return (
-    <ManagementPageView
-      viewType="list"
-      data-test-subj="endpointPage"
-      headerLeft={
-        <>
-          <EuiFlexGroup alignItems="center" responsive={false}>
-            <EuiFlexItem grow={false}>
-              <EuiTitle size="l">
-                <h1 data-test-subj="pageViewHeaderLeftTitle">
-                  <FormattedMessage
-                    id="xpack.securitySolution.endpoint.list.pageTitle"
-                    defaultMessage="Endpoints"
-                  />
-                </h1>
-              </EuiTitle>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiBetaBadge
-                label={i18n.translate('xpack.securitySolution.endpoint.list.beta', {
-                  defaultMessage: 'Beta',
-                })}
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-          <EuiSpacer size="s" />
-          <EuiText size="s" color="subdued">
-            <p>
+    <WrapperPage noTimeline data-test-subj="endpointPage">
```

Comment:
This might be a personal preference, but I would have liked it if it was just one component that included everything that should be presented on a page under the management view, so that we did not have to scaffold up the page instead of us having to create the page manually every time (`<Headerpage>` ++ `<SiemNavigation>` ++ `<EuiSpacer>` ++ `<EuiPanel>`). It would also help to ensure that we remain consistent with the UI across pages as it would all be centralized. One can always revert back to "manual scaffolding" if needed.

What if we kept `<ManagementPageView>` and add all this internal UI component to it, and expose pass-through props as needed? Usage could be something like:

```jsx
<ManagementPageView
  noTimeline
  data-test-sugj="endpointPage"
  headerTitle="Title as passthrough to HeaderPage"
  headerSubtitle="sub title as passthrough to HeaderPage"
  badgeOptions={/* passthrough to HeaderPage */}
>{
    /* content of body here */
}
</ManagementPageView>
```

Reply:
So I've created AdministrationListPage component, which is partial solution to shared component. It's only used for list pages as they have a lot of similarities. Other places were too different to put them all in the same component. Let's have a discussion and effort later to create other ones. I can see some sort of AdministrationDetailsPage, still at this moment there's only one single page like that (policy details) and actually that page is now more related to ingest than administration section. The only other usage of WrapperPage is error page for policy details, which I personally would suggest to completely eliminate in favour of redirect and toast.